app/controllers/app.rb

```diff
@@ -21,36 +25,58 @@ class App < Roda
         view 'home'
       end
 
+      # GET /404
       routing.on '404' do
-        routing.get do
-          view '404'
-        end
+        view '404'
       end
 
+      # GET /resource_nil
       routing.on 'resource_nil' do
         view 'resource_nil'
       end
 
       # Inbox
       routing.on 'inbox' do
-        routing.on 'new' do
+        
+        # GET /inbox/new-inbox
+        routing.on 'new-inbox' do
           view 'new_inbox'
         end
 
+        # GET /inbox/guest-inbox
+        routing.on 'guest-inbox' do
+          
+          # Get guest suggestions
+          session[:guest_suggestions] ||= []
+          
+          session[:guest_suggestions].insert(0, "suggestion1").uniq!
+          session[:guest_suggestions].insert(0, "suggestion2").uniq!
+          # session[:guest_suggestions].delete("suggestion1")
+          # session[:guest_suggestions].delete("suggestion2")
+          puts "guest_suggestions = #{session[:guest_suggestions]}"
+
+          view 'guest_inbox', locals: { guest_suggestions: session[:guest_suggestions] }
```

Comment:
Hey, I think we don't need a 'guest_inbox' view. This is because inbox and guest inbox are the same thing. An inbox shouldn't care if the user is a guest or not. It's just mean to show suggestions and allow them to save or discard them. 

Plus there's less code for us to maintain and keep in sync

Reply:
For **guest-inbox**, I added some new test case in `domain_inboxes_spec.rb`
![image](https://user-images.githubusercontent.com/12498220/99901189-b7f49f00-2cef-11eb-96c9-7b4725f301a8.png)

And make **guest-inbox** and **inbox** reuse the same views.
Now the **guest-inbox** looks like:
![image](https://user-images.githubusercontent.com/12498220/99901237-1588eb80-2cf0-11eb-8442-6b6bf76088f0.png)